src/Essentials/src/Geolocation/Geolocation.android.cs

```diff
@@ -18,24 +19,27 @@ partial class GeolocationImplementation : IGeolocation
 		const long twoMinutes = 120000;
 		static readonly string[] ignoredProviders = new string[] { LocationManager.PassiveProvider, "local_database" };
 
-		static ContinuousLocationListener continuousListener;
-		static List<string> listeningProviders;
+		static ContinuousLocationListener? continuousListener;
+		static List<string>? listeningProviders;
 
-		static LocationManager locationManager;
+		static LocationManager? locationManager;
 
-		static LocationManager LocationManager =>
+		static LocationManager? LocationManager =>
```

Comment:
I'm not sure if LocationManager can ever be null on a sane Android environment, but there might be some weird custom ROMs; I guess the best is to throw `FeatureNotSupportedException` then...

Reply:
@jfversluis there are still my the questions...